test/avro_json_decoder_tests.erl

```diff
@@ -85,12 +85,21 @@ decode_with_default_values_test() ->
                                  , {<<"default">>, null}
                                  , {<<"order">>, <<"ignore">>}
                                  ])
+                     , ?JSON_OBJ([ {<<"name">>, <<"union_null_field">>}
+                                 , {<<"type">>, [<<"null">>, <<"int">>]}
+                                 , {<<"default">>, "null"}
```

Comment:
Should be `<<"null">>` here otherwise this `"null"` is encoded as `"default":[110,117,108,108]` in JSON.
The test case is passing because `[ignore_bad_default_values]` is used in this test case.
Also, the very next line that runs without `ignore_bad_default_value`
which expects a `bad_default` exception is catching another field.